mono/mini/magic-division.h

```diff
@@ -0,0 +1,253 @@
+#ifndef __MONO_MINI_MAGIC_DIVISION_H__
```

Comment:
This is not a header file.

Move this to a .c file or include the content in local-propagation.c.